ArjunaJTA/jee_transactional_app/pom.xml

```diff
@@ -90,20 +64,6 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-war-plugin</artifactId>
-                <version>2.1.1</version>
-                <configuration>
-                    <!-- Java EE 6 doesn't require web.xml, Maven needs to catch up! -->
-                    <failOnMissingWebXml>false</failOnMissingWebXml>
-                    <archive>
-                        <manifestEntries>
-                            <Dependencies>org.jboss.jts</Dependencies>
-                        </manifestEntries>
-                    </archive>
```

Comment:
Isn't this needed (I guess for WFLY?)

Reply:
I tested it with all the profiles (default, arq and arqIPv6) and I found out that such a configuration is not needed in order to have a successful result. Should I keep that configuration anyway @jmfinelli ?

Reply:
If you double check it works without...that's great! Thanks!